lighthouse-core/report/v2/report-generator.js

```diff
@@ -79,9 +80,11 @@ class ReportGeneratorV2 {
   generateReportHtml(reportAsJson) {
     const sanitizedJson = JSON.stringify(reportAsJson).replace(/</g, '\\u003c');
     const sanitizedJavascript = REPORT_JAVASCRIPT.replace(/<\//g, '\\u003c/');
+    const sanitizedCss = REPORT_CSS.replace(/<\//g, '\\u003c/');
```

Comment:
@ebidel I responded to that. This wasn't for XSS, this was for future-proof against us somewhere using `</script>` in a string and scratching heads for why the whole report is broken. We do it [currently](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/report/report-generator.js#L40) too.